### PR TITLE
Integration cards layout

### PR DESCRIPTION
**Pull request by JordanSh**
## Issue
- https://github.com/elastic/security-team/issues/2760

## Summary
This PR mainly focuses on CSS and layout changes.

## Before
image

## After
image

### CODE REVIEW

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx [hunk 1]
```diff
@@ -68,93 +73,71 @@ export const BenchmarksSection = () => {
...
+              </EuiText>
+              {INTERNAL_FEATURE_FLAGS.clusterMetaMock && (
+                <>
+                  <EuiText>
+                    <h4>{`Cluster ID ${mockClusterId}`}</h4>
+                  </EuiText>
+                  <EuiSpacer size="xs" />
+                  <EuiText size="xs" color="subdue">
+                    <EuiIcon type="clock" />
+                    {'Updated 7 second ago'}
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx:90]: do we want this hardcoded?

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx [hunk 2]
```diff
@@ -68,93 +73,71 @@ export const BenchmarksSection = () => {
...
             </EuiFlexItem>
-          </EuiFlexGrid>
+          </EuiFlexGroup>
         </EuiPanel>
       ))}
     </>
   );
 };
+
+const getIntegrationBoxStyle = (euiTheme: EuiThemeComputed) => ({
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx:136]: IMO it'd be preferable if Product used a known UI component so we won't have to add custom styles. 

maybe wrapping in `EuiPanel` would do the trick

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx [hunk 3]
```diff
@@ -68,93 +73,71 @@ export const BenchmarksSection = () => {
...
-                  {
-                    title: TEXT.TOTAL_FAILURES,
-                    description: benchmark.totalFailed || TEXT.ERROR,
-                  },
-                ]}
-              />
+            <EuiFlexItem grow={4}>
+              <ChartPanel
+                title={TEXT.RISKS}
+                hasBorder={false}
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx:122]: is this used with `true` too?

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx (hunk 1 above)
Comment (JordanSh) [reply x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx:90]: personally im ok with it, it's just mock data, won't stay here long. probably deleted by next pr..

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx (hunk 2 above)
Comment (JordanSh) [reply x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx:136]: that would not be enough. it still needs background, border radius changes, remove margins, and such from EuiPanel.. 
in general yeah, we should prioritize using default components like they are.. but this is a pretty rare case I think its fine.

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx (hunk 3 above)
Comment (JordanSh) [reply x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx:122]: yeah in `summary-section.tsx`

**Review by orouz — APPROVED** [review]

**Pull request merged**